### PR TITLE
fix(SelectInput): isLoading vs empty state

### PR DESCRIPTION
**Pull request by lisalupi**
### Concisely:
`SelectInput`: empty state should no longer override `isLoading`.

### Behavior change
When `isLoading` is set, the dropdown now renders a loading skeleton as the sole content. For **ungrouped** options this means `AddOption` / `SelectAll` are no longer shown above the skeleton while loading (previously they were rendered alongside it). This makes ungrouped consistent with grouped, which already showed only the skeleton during loading.

## Type

- Bug

### CODE REVIEW

**Review by etienne-scaleway — APPROVED** [review]

**Review by philibea — APPROVED** [review]

File: packages/ui/src/components/SelectInput/components/Dropdown/Content.tsx [hunk 1]
```diff
@@ -110,6 +110,19 @@ export const CreateDropdown = ({
     </Stack>
   )
 
+  if (isLoading) {
```
Comment (jsulpis) [inline packages/ui/src/components/SelectInput/components/Dropdown/Content.tsx:113]: Behavior change: this early return now renders only `<Skeleton>`, so during loading the **ungrouped** case no longer shows `AddOption`/`SelectAll` (previously they rendered above the skeleton in the ungrouped branch). This makes ungrouped consistent with grouped, but it's still a visible change — can you confirm it's intended and mention it in the PR body?

File: packages/ui/src/components/SelectInput/components/Dropdown/Content.tsx [hunk 2]
```diff
@@ -110,6 +110,19 @@ export const CreateDropdown = ({
...
+    return (
+      <Stack
```
Comment (jsulpis) [inline packages/ui/src/components/SelectInput/components/Dropdown/Content.tsx:115]: `data-grouped` is now dropped during loading: the grouped container still sets `data-grouped`, but this shared loading block doesn't. The snapshot confirms `data-grouped="true"` is removed from the grouped loading case. Anything relying on that attribute while loading?

File: packages/ui/src/components/SelectInput/components/Dropdown/Content.tsx [hunk 3]
```diff
@@ -110,6 +110,19 @@ export const CreateDropdown = ({
...
+        className={selectInputStyle.dropdownContainer}
+        id="select-dropdown"
+        onKeyDown={handleKeyDownSelect}
+        role="listbox"
+      >
+        <Skeleton variant="block" />
```
Comment (jsulpis) [inline packages/ui/src/components/SelectInput/components/Dropdown/Content.tsx:121]: The listbox container has `role="listbox"` but no `aria-busy`/`aria-live` here. Previously the skeleton wrapper carried `aria-busy="true"` and `aria-live="polite"`. An empty listbox that will be populated later should probably stay `aria-busy` so screen readers don't announce an empty list.

File: packages/ui/src/components/SelectInput/components/Dropdown/Content.tsx [hunk 4]
```diff
@@ -183,73 +192,67 @@ export const CreateDropdown = ({
...
-            option={{
-              label: addOptionLabel,
-              searchText: searchInput,
-              value: `${addOption?.text} ${searchInput}`,
-            }}
-            searchable={searchable}
-            textVariant={textVariant}
-          />
-          <SelectAll textVariant={textVariant} />
+      <AddOption
```
Comment (jsulpis) [inline packages/ui/src/components/SelectInput/components/Dropdown/Content.tsx:195]: Nit: the listbox `Stack` wrapper (this block vs the loading early return at ~115–122 and the ungrouped block at ~152–158) and the `AddOption`/`SelectAll` markup are duplicated across the three branches. Extracting the shared wrapper/items into a variable or component would reduce drift risk — the snapshot churn here is largely a symptom of that duplication.

**Review by jsulpis — COMMENTED** [review]
Missing regression test: both existing loading tests use **non-empty** data (`dataGrouped`, `dataUnGrouped`). This PR fixes "empty state overriding isLoading", but that exact scenario (`isLoading` + empty `options={[]}`) isn't covered — please add a test asserting the skeleton renders (not "No options") when `isLoading` with empty data.

**Review by jsulpis — APPROVED** [review]
